apps/public-reference/pages/account/applications.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext, useEffect, useState, Fragment } from "react"
+import Head from "next/head"
+import {
+  ApiClientContext,
+  AppStatusItem,
+  DashBlock,
+  DashBlocks,
+  HeaderBadge,
+  LinkButton,
+  MetaTags,
+  Modal,
+  t,
+} from "@bloom-housing/ui-components"
+import Layout from "../../layouts/application"
+import Archer from "@bloom-housing/listings-service/listings/archer.json"
+import moment from "moment"
+import { Application } from "@bloom-housing/backend-core/client"
+
+export default () => {
+  const { applicationsService } = useContext(ApiClientContext)
+  const [applications, setApplications] = useState([])
+  const [deletingApplication, setDeletingApplication] = useState(null)
+
+  useEffect(() => {
+    // applicationsService.list().then((apps) => {
+    //   setApplications(apps)
+    // })
```

Comment:
Same question as above…my understanding is that we can pull live data from the DB now instead of test data?

Reply:
As we talked on the standup, @bk3c wanted to merge this one sooner than later, so I prepared a separate issue for that.

Reply:
Found it: #532. Sounds good.